sift/client.py

```diff
@@ -12,7 +12,6 @@
 from . import version
 
 API_URL = 'https://api.siftscience.com'
-logging.basicConfig()
 sift_logger = logging.getLogger('sift_client')
```

Comment:
Ok I'll take a look. only thing is I'm running a Django app where I set a LOGGING dictionary in settings so I'm not explicitly calling logging.basicConfig/dictConfig/etc